api/v1alpha3/azuremachine_types.go

```diff
@@ -37,7 +38,14 @@ type AzureMachineSpec struct {
 	VMSize           string           `json:"vmSize"`
 	AvailabilityZone AvailabilityZone `json:"availabilityZone,omitempty"`
 
-	Image *Image `json:"image,omitempty"`
+	// Image is used to provide details of an image to use during VM creation. If set it should be
+	// one of AzureImageByID, AzureSharedGalleryImage or AzureMarketplaceImage.
+	// If image details are omitted the image will default to Ubuntu from the Marketplace.
```

Comment:
That's not technically true, we don't default to the plain Canonical Ubuntu image. We default to the "capi" offer that is built by the "cncf-upstream" publisher which is based on Ubuntu 18.04.

```suggestion
	// If image details are omitted the image will default to the Azure Marketplace "capi" offer, which is based on Ubuntu 18.04.
```

Reply:
Thats a very good point. I've changed it to be inline